rls/deploy/create_rls.py

```diff
@@ -341,9 +332,36 @@ def write_csv(qs_rls, rls_s3_filename):
     upload_to_s3(TMP_RLS_FILE, rls_s3_filename)
 
 
+def set_log_level(RLS_LOGGING_LEVEL):
```

Comment:
This can be done in one line.

`          logger.setLevel(getattr(logging, os.environ.get('LOG_LEVEL', 'INFO').upper(), logging.INFO))`

Reply:
I will leave that function, as I need also protection from a typo in the log level value.